lib/ansible/modules/source_control/github_webhook_facts.py

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/python
+#
+# Copyright: (c) Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+---
+module: github_webhook_facts
+short_description: Query information about Github webhooks
+version_added: "2.6"
+description:
+  - "Query information about Github webhooks"
+requirements:
+  - "PyGithub >= 1.3.5"
```

Comment:
Could you please use `github3.py` instead of `PyGithub` as this will match with other github_* modules ?

Reply:
As I mentioned on the other review, `github3.py` has pretty bad GitHub Enterprise support, so no.

Reply:
I think this will add additional overhead of separate library to both user and maintainer. So -1 from my side.

Reply:
So what would be an appropriate path forward here? `github3.py` is really bad at talking to GitHub Enterprise, so it's clearly not an option. Are you suggesting that I first port the other GitHub-related models to PyGithub in order to merge this?

Blocking this review on the fact that the *other* Github-related modules made a poor choice of library isn't terribly reasonable IMO. So I'd like to know what you suggest to move forward. (Aside from using `github3.py`, of course, since it doesn't work.)

Reply:
I don't see how that's a good choice. It seems to me that the other modules made a poor choice of library, and so going forward we should choose a better one. I've selected PyGithub, but I'm obviously open to other suggestions. What I'm not open to is either a) being required to use the same broken library as the other modules; or b) reinventing the wheel.

Reply:
@adrianmoisey @Akasurde @andreparames @bincyber @dj-wasabi @djmattyg007 @dsummersl @pcgentry @yeukhon

I believe you've worked on other GitHub modules, it would be good to get your views on this

Reply:
I would say then port rest of the github modules using `PyGithub` library so that everyone is benefited. What say ?

Reply:
IMO that sets an unreasonably high barrier to entry, and it's something I have neither the time nor desire to do ATM. I'll leave this PR here in case someone else wants to do it in the future, but it's going to be more appropriate for us to just maintain our own version of this module if that's the cost of getting it merged.

Reply:
@Akasurde I think you're being unreasonable. There is no technical reason for rejecting this patch. The dependencies for each module are clearly indicated in documentation. I don't see how that could be considered confusing. IMO there are far more pressing quality concerns for Ansible's suite of modules than which API wrapper each github module uses.

Reply:
OK I am dismissing my all reviews. Thanks.

Reply:
Making this as resolved.